### ISSUE TITLE
Is there the possibility of having something similar but Fortinet?

### ISSUE THREAD

**Issue by CRAG666** [dlenski/gp-saml-gui#60]
I currently need to connect to Fortinet, so far I do it through copying the cookie but it is tedious.Find your very interesting project.And I'm sure you could have something similar to capture the Cokie for Fortinet.

**Comment by dlenski** [comment]
Yeah, similar to #35 for AnyConnect, and also see [OpenConnect issue #356](https://gitlab.com/openconnect/openconnect/-/issues/356).

If you can write a PR to factor out the protocol-specific parts cleanly, and implement the Fortinet side, that'd be great.
